molo/commenting/templates/comments/comments.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags comments mptt_tags molo_commenting_tags i18n %}
+
+{% block content %}
+
+<div id="comments-list">
+      {% for node in comments %}
+        {% include "comments/comment.html" %}
```

Comment:
This template doesn't exist in this project yet and that's causing your tests to fail